builds/fabric8-che/assembly/assembly-wsagent-war/src/main/java/org/eclipse/che/wsagent/server/WsAgentAnalyticsAddresser.java

```diff
@@ -1,46 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2012-2017 Codenvy, S.A.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *   Codenvy, S.A. - initial API and implementation
- *******************************************************************************/
-package org.eclipse.che.wsagent.server;
```

Comment:
If I remember correctly this file was added here with the idea to change, in the future, the analytics URL to something more related to the RH distribution, or even disable the mechanism totally.

I assume now we automatically get the default one from [here](https://github.com/eclipse/che/blob/master/wsagent/che-wsagent-core/src/main/java/org/eclipse/che/wsagent/server/WsAgentAnalyticsAddresser.java).

@gorkem any idea about this ?